MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -10,6 +10,7 @@ protocol FeedViewControllerDelegate {
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
 	var delegate: FeedViewControllerDelegate?
+	@IBOutlet public weak var errorView: ErrorView!
```

Comment:
This property doesn't need to be `weak`. Also, the setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) public var errorView: ErrorView!
```